main/mariadb/APKBUILD

```diff
@@ -2,9 +2,10 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Contributor: Łukasz Jendrysik <scadu@yandex.com>
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Contributor: TBK <alpine@jjtc.eu>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=mariadb
-pkgver=10.1.26
+pkgver=10.2.8
```

Comment:
Is 10.2.x ABI compatible with 10.1.x?

Reply:
From what I have been able to read it is almost. The differences are revolving around libmysqlclient.so which is now libmariadb.so - https://lists.launchpad.net/maria-discuss/msg04574.html

I will test it later with abi-compliance-checker.